src/Row.ts

```diff
@@ -7,7 +7,7 @@ import { DNode } from '@dojo/widget-core/interfaces';
 import { CellProperties } from './Cell';
 import { HasColumns, ItemProperties } from './interfaces';
 
-import * as rowClasses from './styles/row.css';
+import * as rowClasses from './styles/row.m.css';
```

Comment:
If you do any more refactoring, it might be worth considering that in the widgets repo these files are usually referenced as `css` or `commonCss` etc...